src/door.rs

```diff
@@ -0,0 +1,291 @@
+use super::cfg::{Config, Door, User};
+use super::dos::Templates;
+use super::{LaunchArgs, SysopCmdArgs};
+
+use chrono::Local;
+use fs4::FileExt;
+use serde::Serialize;
+use std::env;
+use std::fs;
+use std::path::Path;
+use std::process::{Child, Command, Stdio};
+
+#[derive(Serialize, Debug)]
+struct LaunchVars<'a> {
+    user: &'a User,
+    node: i8,
+    current_time: String,
+}
+
+#[derive(Serialize, Debug)]
+struct BatchCommands {
+    commands: String,
```

Comment:
I feel like `commands` (plural) shouldn't be a single `String`?

Reply:
It comes from a YAML file, where it may be a single-line or multiline string. It gets interpolated with Handlebars and then appended to the `doorman.bat.hbr` template to create a batch file to launch the door.